Generic_Plugin.php

```diff
@@ -84,6 +84,31 @@ function run() {
 					'ob_callback'
 				) );
 		}
+
```

Comment:
there's still code in general.

The idea is that if you pollute general module - everything will be there at some point and it becomes unmaintantable.

Reply:
I moved the code to a new "UserExperience_Plugin" module.